frontend/host-app/src/components/CreateEventModal/InputHashTag.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {styled} from "@material-ui/core/styles";
+import {TextField} from "@material-ui/core";
+import uuidv1 from "uuid/v1";
+
+const ENTER_KEY = 13;
```

Comment:
```suggestion
const ENTER_KEY_CODE = 13;
```

ENTER_KEY 보다 ENTER_KEY_CODE 가 의미가 좀더 명확한것 같습니다.